mahiru/definitions/interfaces.py

```diff
@@ -165,6 +165,27 @@ def execute_request(
         raise NotImplementedError()
 
 
+class IStepResult:
+    """Contains and manages the outputs of a step.
+
+    Some cleanup is needed after these have been stored, so they're
+    wrapped up in this class so that we can add a utility function.
+
+    Attributes:
+        files: Dictionary mapping workflow items to Path objects, one
+                for each output of the step. Each Path object points
+                to an image file.
+    """
```

Comment:
This suggests there can be multiple outputs, but the `_save_output` method of `PlainDockerDA` only stores one output. Did I miss something?